docs/customer-360/messaging_modernization/index.md

```diff
@@ -0,0 +1,52 @@
+---
+seo:
+  title: Modernize Messaging Workloads
+  description: This ksqlDB recipe shows you how to modernize messaging workloads to move beyond queues and pub/sub
+---
+
+# Modernize Messaging Workloads
+
+Traditional messaging systems like Message Queues (MQs), Enterprise Service Buses (ESBs), and Extract, Transform and Load (ETL) tools have been widely used for decades to handle message distribution and inter-service communication across distributed applications.
+However, they can no longer keep up with the needs of modern applications across hybrid and multi cloud environments for asynchronicity, heterogeneous datasets and high volume throughput.
+Designed as monolithic systems, they are riddled with many challenges: they lack persistence or the ability to efficiently handle highly scalable, efficient and reliable message delivery.
```

Comment:
```suggestion
Designed as monolithic systems, they are riddled with many challenges: they lack persistence and the ability to efficiently handle highly scalable, efficient and reliable message delivery.
```

Reply:
Not sure about it.

Reply:
> efficiently [...] efficient

Maybe some rewording? Sounds repetitive.

Reply:
This text is straight from https://www.confluent.io/use-case/messaging-modernization/

Not that it shouldn't be changed ;)